README.md

```diff
@@ -0,0 +1,10 @@
+# gist-uploader
+
+This is a command line tool that takes a filename and file nickname as arguments.  
+It then uploads the file as a Gist with the provided nickname.  
+The tool tracks the upload progress to the console and provides the URL gist once it's created.  
+
+**To run in terminal:**  
+```javascript
```

Comment:
This is not javascript code, it's `shell`, `sh` or maybe `bash`. You can also don't specify it.